ui/app/utils/openapi-to-attrs.js

```diff
@@ -21,18 +24,22 @@ export const expandOpenApiProps = function(props) {
     } else if (items) {
       editType = items.type + capitalize(type);
     }
+
     let attrDefn = {
       editType,
       helpText: description,
       sensitive: sensitive,
-      label: name || label,
       possibleValues: prop['enum'],
       fieldValue: isId ? 'id' : null,
       fieldGroup: group || 'default',
       readOnly: isId,
       defaultValue: value || null,
     };
 
+    if (label || name) {
+      attrDefn.label = label || capitalize(name) || capitalize(propName);
```

Comment:
Same question here as above - is this no covered via the `form-field` component?

Reply:
It doesn't capitalize things properly, no. And it leaves it snake cased. I'm not sure why.

Reply:
I think that's because we were assigning name as label here - if label is only ever assigned as `displayAttrs.name` or left blank, I think the component falls back to `name` and does the transform from snake_case to humanized text.

Reply:
I tried that and it left it

Reply:
Huh, I wonder if there's a piece missing. Thanks for looking though - everything looks good!